src/vm/types/signatures.rs

```diff
@@ -637,6 +637,24 @@ impl TypeSignature {
         )))
     }
 
+    pub fn max_string_ascii() -> TypeSignature {
```

Comment:
These will be removed in favor of the code here, right? https://github.com/blockstack/stacks-blockchain/pull/2721#discussion_r662449453

Reply:
yes